episodes/11-exercises.md

```diff
@@ -114,33 +142,95 @@ and that assesses their ability to do so.
+Then the content of the challenge is included on the following lines.
+Finally, you can close the fenced div using another line with least 3 colons.
+
+```markdown
+:::::::::::::::::::::::::::::::::::::: challenge
+
+### Challenge Title
+
+Challenge text, code, and other information goes here
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+```
+
+If you also want to include an expandable solution box for the challenge you can 
+add a solution fenced div within the challenge box.
+The format is the same as for a challenge except the tag is `solution` instead. 
+Note the solutions can all be expanded for more accessible reading using the "Expand All Solutions"
+option at the top of each episode.
+
+
+```
+:::::::::::::::::::::::::::::::::::::: challenge
+
+### Challenge Title
+
+Challenge text, code, and other information goes here
 
-Demonstration of how to write an exercise within the lesson infrastructure,
-including how to add a solution box.
+:::::::::::::: solution
+
+### Solution Title
+
+Solution text, code, and other information
+
+:::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+```
+
+For readability, you may want to have the length of the closing lines match the opening lines.
+See in the above how the challenge and the nested solution's closing lines are similar lengths to the their corresponding opening lines.
+For more information about creating exercises see the [Workbench Documentation for Exercises](https://carpentries.github.io/sandpaper-docs/episodes.html#exerciseschallenges).
 
 ::::::::::::::::::::::::::::::::::::::  challenge
 
-## Exercise: formatting exercises in a lesson site (15 minutes)
+## Exercise: Formatting Exercises in a Lesson Site (15 minutes)
 
 Using the approach demonstrated above,
 format the MCQ you designed previously as an exercise in your lesson site.
 
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-Other types of formative assessment:
+Some other types of formative assessment include:
+
+- **Think, pair, share** - Learners _think_ about an answer to a question, _pair_ up
+with a classmate to discuss their answer, and then _share_ out the consensus they came to 
+with the class.
+- **Sticky notes and written feedback** - Learners each get two colors of sticky notes. 
+If they fall behind or run into an issue, they can put up the 
+color that signals they need help.
+If they have completed and exercise or think the instructor could speed up, they put their
+other sticky note up.
+Then at breaks these stickies can turn into written feedback "minute cards", where the learner 
+writes down one thing they were confused about or that could be improved and one thing that went well
+or they really liked learning.
+- **Reflective assessment** - Learners spend time reflecting on what they have learned so far.
+This can be really helpful for "metacognition", where learners think about the process of learning themselves.
```

Comment:
```suggestion
Reflection aids in transferring newly learned concepts into long-term memory and can be really helpful for "metacognition", where learners think about the process of learning themselves.
Reflection exercises can include using what was taught to labelling a diagram, drawing a concept map, free write, fill in a guided workshop. 
```
@anenadic How do you feel about this change?

Reply:
Thanks so much for your feedback @anenadic!  Made some changes based on it, would appreciate another round of review.

I replied in line about my Bloom's hierarchy concerns. Upon reading your summary message I want to add that I think Bloom's is actually much better discussed in the Learning Objectives episode and then just referenced here.

Reply:
Thanks for that addition! Updated above.